corehq/apps/products/fixtures.py

```diff
@@ -18,16 +18,16 @@
 
 
 def product_fixture_generator_json(domain):
-    if SQLProduct.objects.filter(domain=domain).count() == 0:
+    if not SQLProduct.objects.filter(domain=domain).exists():
         return None
 
     fields = filter(lambda x: x != CUSTOM_DATA_SLUG, PRODUCT_FIELDS)
     fields.append('@id')
 
-    custom_fields = CustomDataFieldsDefinition.get_by_domain_and_type(domain, 'ProductFields')
+    custom_fields = get_by_domain_and_type(domain, 'ProductFields')
     if custom_fields:
-        for f in custom_fields.fields:
-            fields.append(CUSTOM_DATA_SLUG + '/' + f.slug)
+        for f in custom_fields['doc']['fields']:
```

Comment:
one subtle change with this is that if the keys don't exist on the doc then this will fail hard (whereas jsonobject will nicely deal with that). will let @esoergel merge as long as we're never worried this data being messy.